stable/horovod/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for deploying Horovod
 name: horovod
-version: "0.1.3"
+version: "0.2.3"
```

Comment:
When bumping the minor version, please set the patch level to 0. So this would become 0.2.0. Also, please remove the quotes, they are not needed.

Reply:
Thanks, updated.